gradle/spotless.gradle

```
@@ -31,3 +31,9 @@ spotless {
 		greclipse().configFile rootProject.files('gradle/spotless.eclipseformat.xml', 'gradle/spotless.groovyformat.prefs')
 	}
 }
+if (System.env['CI'] == null) {
+	tasks.named('spotlessCheck') {
```

```suggestion
	tasks.named('build') {
```
could use `build` as well, this seem most simple way.
Any objection?

---

I have advised against running `spotlessApply` automatically for some time now. 

- https://github.com/diffplug/spotless/issues/1222#issuecomment-1140469080

I want to enable people to do whatever they want in their own projects, but Spotless ought to be developed with its defaults imo.